### ISSUE TITLE
Change `Contribution.date` name and datatype to `datetime` rather than a `date`. 

### ISSUE THREAD

**Issue by larrybabb** [ga4gh/va-spec#240]
While it might seem *helpful* to allow both `date` and `datetime` as permissible content types, I think it would only keep the produced data in non-compatible forms (technically speaking).  

IMO, it is quite reasonable to imagine that contributions (or activities) to happen multiple times in a single day from the same agent/contributor on the same data and for the same `activityType`.  If other's assume that to be fair, then I think it is best to allow for the more specfic (or precise) datetime representation. Those that only use the `date` portion can use the very common practice of defaulting `00:00:000` for the time portion and commuicating that decision to downstream users of their data.

---
Vote up or down if you agree with changing the attribute `Contribution.date` to `Contribution.datetime` and changing the associated datatype to the gks-core primitive datatype from `date` to `datetime`.

_Originally posted by @larrybabb in https://github.com/ga4gh/va-spec/discussions/234#discussioncomment-11399411_

**Comment by mbrush** [comment]
I agree with changing the data type. but not keen on so strictly tying attribute names to a technical data type.  I think  it is fine to keep `date` as the attribute name - and use a dateTime data type that allows time of day if desired.  But I suspect just the date will be sufficient in most cases.  Don’t feel super strongly however.

**Comment by mbrush** [comment]
**Outcome of 2-7-25 Discussion:**
- keep `date` as the attribute name - and use a `dateTime` data type as the value (which allows time of day if desired - but in the vast majority of cases a date is likely to suffice)

**Comment by mbrush** [comment]
Addressed in PR #277
